build-support/bin/generate_docs.py

```diff
@@ -194,7 +199,7 @@ def create_parser() -> argparse.ArgumentParser:
 
 
 def run_pants_help_all() -> Dict:
-    deactivated_backends = ["internal_plugins.releases"]
+    deactivated_backends = ["internal_plugins.releases", "pants.backend.experimental.java"]
```

Comment:
Necessary due to https://github.com/pantsbuild/pants/issues/12889

We should probably run this script (without syncing to the live docs) in CI.